schema.graphql

```diff
@@ -5103,6 +5104,23 @@ enum PostalCodeRuleInclusionTypeEnum {
   EXCLUDE
 }
 
+type PreorderData {
+  isPreorder: Boolean!
```

Comment:
What's the scenario when variant.preorder is not a null and `isPreorder` is false?

Reply:
The BE schema makes `variant.preorder` required, so that case is the equivalent of having `variant.preorder` null.

Reply:
Talked to @maarcingebala and it'll be changed in backend schema as well

Reply:
I think this is already changed

Reply:
and already adjusted in this PR

Reply:
@JanChodorowski then why is the `isPreorder` field still here? Did backend leave it for a purpose?

Reply:
We changed the behavior to return `null` when variant is not in preorder, and to return `PreorderData` object when it is. But the field `isPreorder` was not removed then and it is redundant in this case, we will drop it entirely.

Reply:
@JanChodorowski @bmigirl I've just pushed removal of this `isPreorder` field from API, so this code could be updated now.